ibis/impala/compiler.py

```diff
@@ -125,6 +126,9 @@ def _cast(translator, expr):
 
     if isinstance(arg, ir.CategoryValue) and target_type == 'int32':
         return arg_formatted
+    if (isinstance(arg, (ir.TimeValue, ir.DateValue, ir.TimestampValue)) and
+            target_type == 'int64'):  # noqa E129
```

Comment:
Which `noqa` warning is this? If it's a formatting warning then you should be able to remove it by breaking a line just before the `):`.

Reply:
This PEP8 issue is: `E125 continuation line with same indent as next logical line`. I cannot seem to get the indention right to pass.

Reply:
Worst case you do this:

```python
if (
    expression
):
    # thingz